openapi/weaviate/Ballerina.toml

```diff
@@ -0,0 +1,11 @@
+[package]
+license = ["Apache-2.0"]
+keywords = ["Vector", "Search", "Cost/Freemium", "AI/Weaviate"]
```

Comment:
```suggestion
keywords = [ "AI/Weaviate", "Cost/Freemium", "Vector", "Search"]
```

Reply:
Shall we use vector databases? and vector search or embedding search

Reply:
Addressed in https://github.com/ballerina-platform/openapi-connectors/pull/754/commits/4011e12c5729754ff4de72e28289bb1362275fd0.